presto-client/src/main/java/com/facebook/presto/client/StatementClientV1.java

```diff
@@ -207,6 +207,11 @@ private Request buildQueryRequest(ClientSession session, String query)
             builder.addHeader(PRESTO_SESSION_FUNCTION, urlEncode(entry.getKey()) + "=" + urlEncode(entry.getValue()));
         }
 
+        Map<String, String> customHeaders = session.getCustomHeaders();
+        for (Entry<String, String> entry : customHeaders.entrySet()) {
+            builder.addHeader(entry.getKey(), urlEncode(entry.getValue()));
```

Comment:
what happens if I pass in `PRESTO_SESSION_FUNCTION` as the key ?
will it override what we set in line 207

Reply:
I could not find any method to access the header. I am going to move it at the top so it won't override.